skimage/util/_montage.py

```diff
@@ -94,7 +94,8 @@ def montage(arr_in, fill='mean', rescale_intensity=False, grid_shape=None,
         arr_in = np.asarray(arr_in)[..., np.newaxis]
 
     if arr_in.ndim != 4:
-        raise ValueError('Input array has to be either 3- or 4-dimensional')
+        raise ValueError('Input array has to be 3-dimensional, '
+                         'or 4-dimensional with `multichannel=True`')
```

Comment:
Sorry to nitpick, but the comma should be moved to after 4D, because now it reads as 
`(image.ndim == 3) or (image.ndim == 4 and multichannel)`, rather than `(image.ndim in (3, 4)) and multichannel)`